config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: docker.io/aerospike/aerospike-kubernetes-operator-nightly
-  newTag: 2.1.0-dev
+  newName: docker.io/davi17g/aerospike-kubernetes-operator-nightly
```

Comment:
Is it ok to use davi17g repo instead of pushing the image to aerospike repo itself. So that we don't have to change the name.

Reply:
no,I pushed it by mistake, I'll fix that